package.json

```diff
@@ -33,7 +33,9 @@
     "zone.js": "^0.8.4"
   },
   "devDependencies": {
-    "@ionic/app-scripts": "1.2.4",
+    "@ionic/app-scripts": "1.3.2",
+    "@ionic/cli-plugin-cordova": "0.0.12",
```

Comment:
@gianpaj They're needed for CLI v3: https://github.com/driftyco/ionic-cli/tree/v3#install